Generator.Equals/Models/AttributesMetadata.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace Generator.Equals.Models;
+
+internal sealed record AttributesMetadata(
+    AttributeMetadata Equatable,
+    AttributeMetadata DefaultEquality,
+    AttributeMetadata OrderedEquality,
+    AttributeMetadata IgnoreEquality,
+    AttributeMetadata UnorderedEquality,
+    AttributeMetadata ReferenceEquality,
+    AttributeMetadata SetEquality,
+    AttributeMetadata StringEquality,
+    AttributeMetadata CustomEquality,
+    ImmutableDictionary<long, string> StringComparisonLookup)
```

Comment:
`ImmutableDictionary` is also not equatable.

Reply:
Technically yes but i accidently use AttributesMetadata as a singleton. Nontheless i fixed it as you suggested.